packages/build/src/source/resource/models/ResourceList.ts

```diff
@@ -0,0 +1,15 @@
+
+export default class ResourceList
+{
+    readonly #resources: string[];
+
+    constructor(resources: string[])
+    {
+        this.#resources = resources;
+    }
+
+    isModuleResource(moduleFilename: string): boolean
```

Comment:
In this context, I find the following order more clear. Just flushing my thoughts, ignore if you don't agree.

```suggestion
    isResourceModule(moduleFilename: string): boolean
```